x-pack/plugins/fleet/public/applications/integrations/sections/epm/components/package_list_grid.tsx

```diff
@@ -128,37 +124,65 @@ export const PackageListGrid: FunctionComponent<Props> = ({
     <>
       {featuredList}
       <div ref={menuRef}>
-        <EuiFlexGroup alignItems="flexStart" gutterSize="xl">
+        <EuiFlexGroup
+          alignItems="flexStart"
+          gutterSize="xl"
+          data-test-subj="epmList.integrationCards"
+        >
           <EuiFlexItem grow={1} className={isSticky ? 'kbnStickyMenu' : ''}>
             {controlsContent}
           </EuiFlexItem>
           <EuiFlexItem grow={5}>
-            <EuiSearchBar
-              query={searchTerm || undefined}
-              box={{
-                'data-test-subj': 'epmList.searchBar',
-                placeholder: i18n.translate('xpack.fleet.epmList.searchPackagesPlaceholder', {
-                  defaultMessage: 'Search for integrations',
-                }),
-                incremental: true,
-              }}
-              onChange={onQueryChange}
-              toolsRight={
+            <EuiFieldSearch
+              data-test-subj="epmList.searchBar"
+              placeholder={i18n.translate('xpack.fleet.epmList.searchPackagesPlaceholder', {
+                defaultMessage: 'Search for integrations',
+              })}
+              value={searchTerm}
+              onChange={(e) => onQueryChange(e)}
+              isClearable={true}
+              incremental={true}
+              fullWidth={true}
+              prepend={
                 selectedCategoryTitle ? (
-                  <div>
-                    <EuiBadge
-                      color="accent"
-                      iconType="cross"
-                      iconSide="right"
-                      iconOnClick={() => {
+                  <EuiText
+                    data-test-subj="epmList.categoryBadge"
+                    size="xs"
+                    style={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      fontWeight: euiTheme.font.weight.bold,
+                      backgroundColor: euiTheme.colors.lightestShade,
+                    }}
+                  >
+                    <EuiScreenReaderOnly>
+                      <span>Searching category: </span>
+                    </EuiScreenReaderOnly>
+                    {selectedCategoryTitle}
+                    <button
+                      data-test-subj="epmList.categoryBadge.closeBtn"
+                      onClick={() => {
+                        setSearchTerm('');
```

Comment:
I wonder if we should leave the search term in place when we remove the category. Seems like clearing both the query and the category is a little unexpected when "toggling off" the category via this button.

Reply:
I added this control to try avoiding [this bug](https://github.com/elastic/kibana/issues/140155):
basically if you remove the category while there's text on it and don't clear the query, there is going to be no search until another keystroke, but that's how the `euiSearchField` works (it expects an input to trigger a search). 

I'll remove this line and add a comment in the bug.

Reply:
Ah got it - if this avoids a bug I'm fine leaving it with a comment about the issue w/ `EuiSearchField`. Thanks for clarifying.

Reply:
Well I was also unsure about it. Clearing the query when removing the category can be unexpected, so I'll remove it and I'll address the bug separately in the ticket.